perun-wui-core/src/main/java/cz/metacentrum/perun/wui/model/beans/Service.java

```diff
@@ -46,6 +57,51 @@ public final native String setName(String name)  /*-{
         this.name = name;
     }-*/;
 
+	/**
+	 * Set description of Service
+	 *
+	 * @param description
+	 */
+	public final native void setDescription(String description)/*-{
+    	this.description = description;
+	}-*/;
+
+	/**
+	 * Set Delay
+	 *
+	 * @param delay
+	 */
+	public final native void setDelay(int delay)/*-{
```

Comment:
Getters for new properties are missing. Please note, that in most object getters for Id,Name and Description are not present, since they are in a parent implementation (shared between most perun beans).